ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -117,14 +119,6 @@
                       @models={{array @backend metadata.fullSecretPath}}
                     />
                   {{/if}}
-                  {{#if metadata.canCreateVersionData}}
```

Comment:
Why was this one removed?

Reply:
Good question, I should have left a comment! With the addition of the `Overview` link, design felt the popup menu was too long and that we could remove this action.